chaosaws/ecs/probes.py

```diff
@@ -1,2 +1,22 @@
 # -*- coding: utf-8 -*-
-__all__ = []
+from chaoslib.types import Configuration, Secrets
+
+from chaosaws import aws_client
+
+__all__ = ["ecs_service_is_not_deploying"]
+
+
+def ecs_service_is_not_deploying(cluster: str,
+                                 service: str,
+                                 configuration: Configuration = None,
+                                 secrets: Secrets = None) -> bool:
+    """
+    Checks to make sure there is not an in progress deployment
+    """
+    client = aws_client("ecs", configuration, secrets)
+    response = client.describe_services(
+        cluster=cluster,
+        services=[service]
+    )
+
+    return len(response['services']['deployments']) == 1
```

Comment:
Is there a possibility one of those two keys do not appear in the response? If that is the case, this code will break I think.

Reply:
Yes, if the request results in an error, the `deployments` key will not exist. I'll fix that

Reply:
This is fixed in the latest commit